ReShade.vcxproj

```diff
@@ -368,7 +368,6 @@
     <ClCompile Include="source\d3d11\state_block.cpp" />
     <ClCompile Include="source\d3d11\draw_call_tracker.cpp" />
     <ClCompile Include="source\d3d12\d3d12.cpp" />
-    <ClCompile Include="source\d3d12\d3d12_command_list.cpp" />
```

Comment:
Please don't remove `d3d12_command_list.hpp` and `d3d12_command_list.cpp` yet. They may become useful in the future, since I plan on moving the function hooks into those (after fixing the crash you were seeing witht that). It's also not really related to Vulkan.

Reply:
I reintroduced them, bit i was forced to remove the ID3D12GraphicsCommandList5 API methods (because of the d3d12.h lib used).